packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -232,9 +235,13 @@ export class EmberishCurlyComponent extends GlimmerObject {
   public attrs: Attrs;
   public element: Element;
   public parentView: Component = null;
+  public args: ProcessedArgs;
+  public argsRevision: number;
 
   static create(args: { attrs: Attrs }): EmberishCurlyComponent {
-    return super.create(args) as EmberishCurlyComponent;
+    let result = super.create(args) as EmberishCurlyComponent;
+    result.argsRevision = result.args && result.args.tag && result.args.tag.value();
```

Comment:
We also have another problem, when we did the work to add tags, we didn't bother to implement it correctly in glimmer-objects since they are "only used in tests", so pretty much EVERYTHING in the test environment is volatile 😞 😞 😞 😞 😞

So even if you add this here chances are that it would probably just always invalidate 😐 This is actually quite problematic, because we use the same setup in benchmarks. It just so happens that most of the benchmarks we are doing wouldn't be able to take advantage of those optimizations anyway, but still...